flocker/node/_deploy.py

```diff
@@ -80,11 +80,17 @@ class IDeployer(Interface):
     changes to bring local state and desired cluster configuration into
     alignment.
     """
-    def discover_local_state():
+    def discover_local_state(local_state):
```

Comment:
This impacts the caller.  Can you add something about what changes this will require in `_loop.py`?